index.d.ts

```diff
@@ -396,21 +432,65 @@ declare module 'snowflake-sdk' {
          */
         arrayBindingThreshold?: number;
 
+        /**
+         * The max login timeout value. This value is either 0 or over 300.
+         */
+        retryTimeout?: number;
+
+        /**
+          * The option to skip the SAML URL check in the Okta authentication
+          */
+        disableSamlUrlCheck?: boolean;
+
+        /**
+          * The option to fetch all the null values in the columns as the string null.
+          */
+        representNullAsStringNull?: boolean;
+
+        /**
+         * Number of threads for clients to use to prefetch large result sets. Valid values: 1-10.
+         */
+        resultPrefetch?: number;
+
+
+        //Connection options Options but not on the web document.
         /**
          * Set whether the retry reason is included or not in the retry url.
          */
         includeRetryReason?: boolean;
 
         /**
-         * The max login timeout value. This value is either 0 or over 300.
+         * Number of retries for the login request.
          */
-        retryTimeout?: number;
+        sfRetryMaxLoginRetries?: number,
+
+        forceStageBindError
```

Comment:
this does not seem to have a type?

Reply:
I did not finish the implementation because I am not sure these options should be in the d.ts file. I finished it, and let me know if we need these options.